src/dialogxml/widgets/scrollbar.cpp

```diff
@@ -86,6 +90,189 @@ eScrollStyle cScrollbar::getStyle() {
 	return style;
 }
 
+// Translate raw x/y position using the view of the current rendering target
+location cScrollbar::translated_location (sf::Vector2i const point) const {
+	return location { this->inWindow->mapPixelToCoords(point) };
+}
+
+bool cScrollbar::handle_event (sf::Event const & event) {
+	// Not visible -> not interested
+	if (!this->isVisible())
+		return false;
+	
+	// Visible but no maximum -> not interested
+	if (this->getMaximum() == 0) {
+		return false;
+	}
+		
+	switch (event.type) {
+		case sf::Event::MouseButtonPressed:
+			return this->handle_mouse_pressed(event);
+		case sf::Event::MouseMoved:
+			return this->handle_mouse_moved(event);
+		case sf::Event::MouseButtonReleased:
+			return this->handle_mouse_released(event);
+		case sf::Event::MouseWheelScrolled:
+			return this->handle_mouse_wheel_scrolled(event);
+		default: break;
+	}
+
+	return false;
+}
+
+bool cScrollbar::handle_mouse_wheel_scrolled (sf::Event const & event) {
+	location event_location = this->translated_location({
+		event.mouseWheelScroll.x,
+		event.mouseWheelScroll.y
+	});
+	
+	// Scrolling outside of catching area -> not interested
+	if (!event_location.in(this->mousewheel_event_catching_rect))
```

Comment:
I think this should check both the "catching" / "controlled" rect and the rect of the scrollbar itself. Obviously the former rect can generally be arranged to encompass the latter rect, but in case for some reason it isn't (maybe it's being used as a slider instead of a scrollbar), using the wheel while hovering the bar should probably still work.

Reply:
done